src/worker/generate-metadata.ts

```diff
@@ -0,0 +1,18 @@
+interface Metadata {
+  license: string;
+  languages: string[];
+}
+
+/**
+ * Given languages, generate a .kmp json file which contains metadata for .kmp zip archive.
+ */
+export function generateMetadata(languages: string[]): string {
+  const license = "mit";
+  const metadata: Metadata = { license, languages };
+
+  return JSON.stringify(metadata);
+}
+
+export function generateMetadataJson(content: string): Blob {
+  return new Blob([content], { type: "application/json" });
+}
```

Comment:
> Because `createZipWithFiles()` takes in strings and not blobs, funnily enough, you don't need to even generate a blob to solve #51!
> 
> See:
> https://github.com/eddieantonio/predictive-text-studio/blob/494fde2eed2964f6315bb5d38f85dbdebc4deedf/src/worker/generate-zip.ts#L8-L18

But don't you need a blob to actually download it?

Reply:
Yes, but the blob is for the `.kmp` file (zip file) — not the JSON.

Reply:
Okay, I see what you mean now